watchtower/src/main.rs

```diff
@@ -323,11 +323,13 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 let total_current_stake = vote_accounts
                     .current
                     .iter()
-                    .fold(0, |acc, vote_account| acc + vote_account.activated_stake);
+                    .map(|vote_account| vote_account.activated_stake)
```

Comment:
@mvines, cluster_query.rs looks to implement almost the exact same code. Maybe cluster_query.rs belongs in solana_client?

Reply:
yeah, no opposed at all to starting to start breaking out business logic parts of cli/. 

`solana_cli::display::write_transaction()` is another example of a dirty deed